pkg/domain/domain_sysvars.go

```diff
@@ -150,3 +154,9 @@ func (do *Domain) changeSchemaCacheSize(ctx context.Context, size uint64) error
 	do.infoCache.Data.SetCacheCapacity(size)
 	return nil
 }
+
+func (do *Domain) changePDMetadataCircuitBreakerErrorRateThresholdPct(errorRatePct uint32) {
+	tikv.ChangePDRegionMetaCircuitBreakerSettings(func(config *circuitbreaker.Settings) {
```

Comment:
Seems this method does not need the `Domain` state, why not just call `tikv. ChangePDRegionMetaCircuitBreakerSettings` in `SetGlobal` ?